dart/lib/src/default_integrations.dart

```diff
@@ -33,3 +36,31 @@ class RunZonedGuardedIntegration extends Integration {
     options.sdk.addIntegration('runZonedGuardedIntegration');
   }
 }
+
+/// integration that runs other integrations
+class NoZonedGuardedIntegration extends Integration {
```

Comment:
based on this assumption https://github.com/getsentry/sentry-dart/pull/280#discussion_r563578375
I don't see why we'd need that?

Reply:
fits your sentence:

> When there is none, all integrations are run inside NoZonedGuardedIntegration. This however seems redundant and can just be a for loop.